### PR TITLE
Make remaining Quickstart docs consistent

### PR DESCRIPTION
**Pull request by SD-13**
**What type of PR is this:**
/area documentation
/kind documentation

**What does this PR do / why we need it:**

**Which issue(s) this PR fixes:**
Fixes part of #6143 

**PR acceptance criteria:**

- [ ] Unit test 

- [ ] Integration test 

- [X] Documentation 

**How to test changes / Special notes to the reviewer:**

### CODE REVIEW

File: docs/website/docs/user-guides/quickstart/dotnet.md [hunk 1]
```diff
@@ -16,9 +16,12 @@ ASP.NET MVC is a web application framework that implements the model-view-contro
 1. Generate an example project:
 
 ```console
-dotnet new mvc --name app
+dotnet new mvc --name <name>
```
Comment (valaparthvi) [inline docs/website/docs/user-guides/quickstart/dotnet.md:19]: I'd keep this as is.

**Review by valaparthvi — CHANGES_REQUESTED** [review]
Everything looks good, except one comment that I've posted.

File: docs/website/docs/user-guides/quickstart/dotnet.md (hunk 1 above)
Comment (SD-13) [reply docs/website/docs/user-guides/quickstart/dotnet.md]: Done.

**Comment by SD-13** [comment]
Hi @valaparthvi , I have pushed your suggested change, please take another look when you are free. Thanks,

**Comment by valaparthvi** [comment]
/lgtm
/approve

Thank you for working on this @SD-13!

**Comment by valaparthvi** [comment]
/override windows-integration-test/Windows-test
It's a Docs PR.

**Pull request merged**